Dockerfile

```diff
@@ -1,32 +1,33 @@
-FROM jupyter/base-notebook:27ba57364579
+FROM jupyter/base-notebook:814ef10d64fb
+# Built from... https://hub.docker.com/r/jupyter/base-notebook/
+#               https://github.com/jupyter/docker-stacks/blob/master/base-notebook/Dockerfile
+# Built from... Ubuntu 18.04
 
-# conda/pip/apt install additional packages here, if desired.
-USER root
-RUN apt-get update && \
-    apt-get install -y --no-install-recommends \
-      git \
-      && \
-    apt-get purge && apt-get clean && \
-    apt-get -y --allow-unauthenticated install vim build-essential wget gfortran bison libibverbs-dev libibmad-dev libibumad-dev librdmacm-dev libmlx5-dev libmlx4-dev graphviz gcc make
```

Comment:
With the image update, we should be able to remove this line, since that's already happened in the baseimage.

Reply:
@NickolausDS I misunderstood this comment and thought it was already part of the PR. [The build is failing](https://hub.docker.com/repository/registry-1.docker.io/rpwagner/jupyter/builds/1b5d9c38-5a96-4c8e-91f9-01e9851ae738) and I think these lines are the cause. Can you do another PR removing anything that's unnecessary?

Reply:
It looks like it's failing for a couple reasons:
* A line above deletes the apt cache before the next apt install runs
* The new image is based on bionic, not the xenial, so a couple packages are missing.
```
Step 7/17 : RUN apt-get -y --allow-unauthenticated install vim build-essential wget gfortran bison libibverbs-dev libibmad-dev libibumad-dev librdmacm-dev libmlx5-dev libmlx4-dev graphviz gcc make
 ---> Running in aa034592fb7a
Reading package lists...
Building dependency tree...
Reading state information...
E: Unable to locate package libmlx5-dev
E: Unable to locate package libmlx4-dev
```

Reply:
Those are Mellanox InfiniBand driver packages. I can't see why we would need them, unless some application we're installing require them.

Reply:
I searched for a little while, and wasn't able to find updated versions of those packages. The PR you just merged does build (at least locally on my machine), so hopefully we can just leave them out.